### ISSUE TITLE
ndless r2019, CX after OS 4.5.4.48 update + ndless_installer,ndless_resources install failed ?

### ISSUE THREAD

**Issue by bo911** [ndless-nspire/Ndless#395]
There was an older ndless + tools installed => No Problem..
After update to OS 4.5.4.48 + ndless_installer-4.5.3.14- 4.5.4.48.tns + ndless_resources.tns + reset
The installer of ndless claims that ndless_resources.tns is not found, but via Filemanager + tilp I saw this file in the ndless - Folder ?

**Comment by Vogtinator** [comment]
This is the blue message at the top of the screen, right? In that case the installer itself ran properly, but failed to open resources for some reason.

This is on a CX non-CAS?

**Comment by bo911** [comment]
I think my error seems to be identical to #343, except I got a CX OS is 4.5.4.48 ... 
After reading here I reset the device (reset button), and try again..
=> The installer start, then the device is reboot before finished... (I thought this come perhaps from nover in ndless/startup, but after renaming the subfolder to ndless/startupp, first the error with not finding ndless_resources.tns come, after reboot try again the installer starts and before correctly finishing => reboot ...
Just to confirm if I download the correct files, I try the same on firebird (linux) => everything worked as exspected ! ?
Is there any command to erase the whole flash to do a fresh reinstalling of OS 4.8.4.48, just to have an reproduceable starting point ?
Any suggestion(s) is welcome !

**Comment by Vogtinator** [comment]
> Is there any command to erase the whole flash to do a fresh reinstalling of OS 4.8.4.48, just to have an reproduceable starting point ?

Yes, using the boot2 maintenance menu you can select "Format filesystem".

**Comment by bo911** [comment]
Sorry, how to enter the boot2 maintenance menu ? I'm at work, so I can proceed at about 19:00...

**Comment by Vogtinator** [comment]
Hold Doc+Enter+EE while resetting the calculator.

**Comment by bo911** [comment]
Thanks, I will post the result(s) !

**Comment by goncalot87** [comment]
I installed the new OS 6.0.3.374 and i cant install Ndless on my TInspire CX II T is there any way of getting the old os back?

**Comment by ExpertPhisherman** [comment]
> I installed the new OS 6.0.3.374 and i cant install Ndless on my TInspire CX II T is there any way of getting the old os back?

no

**Comment by PinkHatGithub** [comment]
Not as of this moment noSent from my iPhoneOn Oct 28, 2023, at 06:45, goncalot87 ***@***.***> wrote:﻿
I installed the new OS 6.0.3.374 and i cant install Ndless on my TInspire CX II T is there any way of getting the old os back?
